tests/fontmap/fontmaptest.cpp

```diff
@@ -26,29 +26,7 @@
 // test class
 // ----------------------------------------------------------------------------
 
-class FontMapperTestCase : public CppUnit::TestCase
-{
-public:
-    FontMapperTestCase() { }
-
-private:
-    CPPUNIT_TEST_SUITE( FontMapperTestCase );
-        CPPUNIT_TEST( NamesAndDesc );
-    CPPUNIT_TEST_SUITE_END();
-
-    void NamesAndDesc();
-
-    wxDECLARE_NO_COPY_CLASS(FontMapperTestCase);
-};
-
-// register in the unnamed registry so that these tests are run by default
-CPPUNIT_TEST_SUITE_REGISTRATION( FontMapperTestCase );
-
-// also include in its own registry so that these tests can be run alone
-CPPUNIT_TEST_SUITE_NAMED_REGISTRATION( FontMapperTestCase, "FontMapperTestCase" );
-
-
-void FontMapperTestCase::NamesAndDesc()
+TEST_CASE("Names And Desc", "[fontmapper]")
```

Comment:
I'd rather avoid using spaces in the test names as this makes it less convenient to run them from command line (they need to be quoted). And it is also convenient to use common prefix for the related tests. So maybe something like this:

```suggestion
TEST_CASE("wxFontMapper::NamesAndDesc", "[fontmapper]")
```